frontend/src/main/web/src/components/Base/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react'
+import warning from 'warning'
+import { merge } from 'lodash'
+import { flattenThemeClasses } from '../../utils/styleUtils'
+
+const Base = ({
+  atomic = {},
+  children,
+  className,
+  componentName,
+  states,
+  tagName,
+  theme = {},
+  ...props
+}) => {
+  const Component = tagName || 'div'
+  if (states && theme.states) {
+    const themeStates = theme.states
+    delete theme.states
```

Comment:
From what I understand, this is to merge `states` into theme if `states` exist. Luke put it in there for just in case the `states` is being used (as attribute) to overrides styling.

From what I can see, `states` is never being used as attribute across the app, which makes this if statement always false.